config/longrun_configs/amip_edonly.yml

```diff
@@ -13,11 +12,11 @@ energy_check: false
 insolation: "timevarying"
 mode_name: "amip"
 netcdf_interpolate_z_over_msl: true
-prescribe_ozone: true
 prescribed_aerosols: ["CB1", "CB2", "DST01", "OC1", "OC2", "SO4", "SSLT01"]
 rmse_check: true
 start_date: "20100101"
 surface_setup: "PrescribedSurface"
 t_end: "366days"
+time_varying_trace_gases: ["CO2", "O3"]
```

Comment:
same here, remove this. (This doesn't do anything, but we can remove it as it is duplicated.)